### PR TITLE
Minor cleanups in parquet

### PR DESCRIPTION
**Pull request by raunaqmorarka**
## Description
Minor cleanups in parquet

## Release notes

(x) This is not user-visible or is docs only, and no release notes are required.
( ) Release notes are required. Please propose a release note for me.
( ) Release notes are required, with the following suggested text:

```markdown
# Section
* Fix some things. ({issue}`issuenumber`)
```

### CODE REVIEW

**Review by wendigo — APPROVED** [review]

**Review by sopel39 — APPROVED** [review]

File: plugin/trino-delta-lake/src/test/java/io/trino/plugin/deltalake/TestDeltaLakeAlluxioCacheFileOperations.java [hunk 1]
```diff
@@ -98,21 +98,21 @@ public void testCacheFileOperations()
                         .add(new CacheOperation("Alluxio.readCached", "00000000000000000002.json", 0, 636))
```
Comment (sopel39) [inline plugin/trino-delta-lake/src/test/java/io/trino/plugin/deltalake/TestDeltaLakeAlluxioCacheFileOperations.java:98]: why all these below went up?

File: lib/trino-parquet/src/main/java/io/trino/parquet/reader/SimpleSliceInputStream.java [hunk 2]
```diff
@@ -140,57 +139,27 @@ public int getByteArrayOffset()
         return offset + slice.byteArrayOffset();
     }
 
-    public void ensureBytesAvailable(int bytes)
-    {
-        checkPositionIndexes(offset, offset + bytes, slice.length());
-    }
-
-    /**
-     * Always check if needed data is available with ensureBytesAvailable method.
```
Comment (sopel39) [inline lib/trino-parquet/src/main/java/io/trino/parquet/reader/SimpleSliceInputStream.java:149]: Is the comment no longer viable? Maybe remove suffix then?

File: plugin/trino-delta-lake/src/test/java/io/trino/plugin/deltalake/TestDeltaLakeAlluxioCacheFileOperations.java (hunk 1 above)
Comment (raunaqmorarka) [reply plugin/trino-delta-lake/src/test/java/io/trino/plugin/deltalake/TestDeltaLakeAlluxioCacheFileOperations.java:98]: started populating 2 new fields is_exact_min/max in the parquet footer

File: lib/trino-parquet/src/main/java/io/trino/parquet/reader/SimpleSliceInputStream.java (hunk 2 above)
Comment (raunaqmorarka) [reply lib/trino-parquet/src/main/java/io/trino/parquet/reader/SimpleSliceInputStream.java:149]: suffix `unchecked` is copied from Slice

**Pull request merged**